docs/security/apparmor.md

```diff
@@ -67,9 +73,111 @@ explicitly specifies the default policy:
 $ docker run --rm -it --security-opt apparmor:docker-default hello-world
 ```
 
+## Loading and Unloading Profiles
+
+To load a new profile into AppArmor, for use with containers:
+
+```
+$ apparmor_parser -r -W /path/to/your_profile
+```
+
+Then you can run the custom profile with `--security-opt` like so:
+
+```bash
+$ docker run --rm -it --security-opt apparmor:your_profile hello-world
+```
+
+To unload a profile from AppArmor:
+
+```bash
+# stop apparmor
+$ /etc/init.d/apparmor stop
+# unload the profile
+$ apparmor_parser -R /path/to/profile
+# start apparmor
+$ /etc/init.d/apparmor start
+```
+
+## Debugging AppArmor
+
+### Using `dmesg`
+
+Here are some helpful tips for debugging any problems you might be facing with
+regard to AppArmor.
+
+AppArmor sends quite verbose messaging to `dmesg`. Usually an AppArmor line
+will look like the following:
+
+```
+[ 5442.864673] audit: type=1400 audit(1453830992.845:37): apparmor="ALLOWED" operation="open" profile="/usr/bin/docker" name="/home/jessie/docker/man/man1/docker-attach.1" pid=10923 comm="docker" requested_mask="r" denied_mask="r" fsuid=1000 ouid=0
+```
+
+In the above example, the you can see `profile=/usr/bin/docker`. This means the
+user has the `docker-engine` (Docker Engine Daemon) profile loaded. On version
```

Comment:
Perhaps we should change the last sentence into a "note"? i.e. for Ubuntu 12.04 ..